tidy3d/components/simulation.py

```diff
@@ -1467,6 +1478,7 @@ def eps_bounds(self, freq: float = None) -> Tuple[float, float]:
         medium_list = [self.medium] + list(self.mediums)
         medium_list = [medium for medium in medium_list if not isinstance(medium, PECMedium)]
         # regular medium
+        # wouldn't using eps_diagonal be more accurate?
```

Comment:
remove comment if it's addressed. otherwise, guess we should discuss more

Reply:
fixed